setup.py

```diff
@@ -26,6 +26,9 @@
     include_package_data=True,
     zip_safe=False,
     platforms='any',
+    install_requires=[
+        'futures;python_version=="2.7"',
```

Comment:
why is it moved to required deps?

Reply:
It's just more modern and semantic correctly way to describe conditional requirements https://www.python.org/dev/peps/pep-0508/#environment-markers

Reply:
I understand the environment part, but was it previously an optional dep for py27, or also required?  If someone is not using Tornado, do they still need this dep?

Reply:
No, it wasn't optional but required conditional dependency for python 2.7. This is how `wheel` deal with conditional requirements:

> Separate ‘extra’ names from conditions using the : character. Wheel’s own setup.py does this. (The empty-string extra is the same as install_requires.) These conditional requirements should work the same whether the package is installed by wheel or by setup.py.

https://github.com/pypa/wheel/blob/master/docs/news.rst at 0.24.0 (2014-07-06)